ros/src/simulation/gazebo_simulator/worlds/gazebo_world_description/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@ project(gazebo_world_description)
 find_package(catkin REQUIRED)
 
 catkin_package()
+add_custom_target(integrate_model ALL /bin/bash ${CMAKE_CURRENT_SOURCE_DIR}/scripts/integrate_model_files.sh)
```

Comment:
the problem is here. you're downloading the files, putting them in `/tmp`, and then moving them to an arbitrary location.

Instead of installing them directly.